server/config/urls.py

```diff
@@ -18,4 +19,5 @@
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
+    url(r'^index/', TemplateView.as_view(template_name='index.html')),
```

Comment:
@ranmoshe There is no `index.html` in the server side. So, let's change the test a bit. The main page of the server side is the main page of the 'Django REST Framework', so let's check if we get 200 there..
The path to the main page of 'Django REST Framework' is `../api`.

So, change it to check that path, and all the other stuff to reflect that the test is testing that page and not `index.html`.